test-workloads/src/bin/tellus.rs

```diff
@@ -109,7 +113,7 @@ fn exercise_pmu_functionality() {
     let start_flags = PmuCounterStartFlags::default();
     let result = start_counters(counter_index, 0x1, start_flags, 0);
     if result.is_err() && !matches!(result.err().unwrap(), sbi::Error::AlreadyStarted) {
-        result.expect("start_counters failed with result {result:?}");
```

Comment:
This condition can be simplified. Also `expect()` will print the error, so you can leave it as a `panic!` or drop the printing of `result`.

```
if !matches!(result, Ok(_) | Err(sbi::Error::AlreadyStarted)) {
    result.expect("start_counters failed");
}
```